src/meson.build

```diff
@@ -0,0 +1,44 @@
+sources = files(
+    'abi_compat.c',
+    'error.c',
+    'names.c',
+    'netplan.c',
+    'networkd.c',
+    'nm.c',
+    'openvswitch.c',
+    'parse.c',
+    'parse-nm.c',
+    'sriov.c',
+    'types.c',
+    'util.c',
+    'validation.c')
+
+libnetplan = library(
+    'netplan',
+    sources,
+    gnu_symbol_visibility: 'hidden',
+    link_args: ['-T', '../abicompat.lds'],
```

Comment:
Thanks!

This doesn't really work, as it will fail with `src/meson.build:16:0: ERROR: File -T does not exist.`, so I need to keep it as `link_args`, but I guess I could use a `linker_script = meson.project_source_root() / 'abicompat.lds'` variable to use both in link_args and link_depends, to make meson aware of any changes relevant to the linker.

Reply:
That's what I mean? You pushed a revision to this PR with the following diff:
```diff
 gnu_symbol_visibility: 'hidden',
-link_depends: '../abicompat.lds',
+link_args: ['-T', '../abicompat.lds'],
 dependencies: [glib, gio, yaml, uuid],
```

Reply:
Oh, indeed! Giving it a 2nd read, I now understand what you meant in the first place. This is as implemented and merged.
So, thank you very much for your input!